pyro/contrib/gp/models/gpr.py

```diff
@@ -39,7 +39,7 @@ class GPRegression(GPModel):
         f &\sim \mathcal{GP}(0, k(X, X)),\\
         y & \sim f + \epsilon,
 
-    where :math:`\epsilon` is noise.
+    where :math:`\epsilon` is a Gaussian noise.
```

Comment:
it needs to be clear (line 59) whether `noise` is variance or square root variance

Reply:
Yes, I changed it too "Variance of Gaussian noise of this model."